mri/operators/fourier/utils.py

```diff
@@ -248,15 +248,16 @@ def generate_operators(data, wavelet_name, samples, mu=1e-06, nb_scales=4,
 def get_stacks_fourier(kspace_loc, shape):
     """Function that splits an incoming 3D stacked k-space samples
     into a 2D non-Cartesian plane and the vector containing the z k-space
-    values of all the plane and converts to stacks of 2D. This function also
-    checks for any issues of the incoming k-space pattern and if the stack
-    property is not satisfied.
+    values of the stacks acquiered and converts to stacks of 2D.
+    This function also checks for any issues of the incoming k-space
+    pattern and if the stack property is not satisfied.
     Stack Property: The k-space locations originate from a stack of 2D samples.
 
     Parameters
     ----------
     kspace_plane_loc: np.ndarray
```

Comment:
The parameter name is `kspace_loc` not `kspace_plane_loc` which actually corresponds to the acquired k-space samples position in the 3 dimension.